packages/machine/src/ethereum/virtual-app-set-state-commitment.ts

```diff
@@ -31,7 +31,7 @@ export class VirtualAppSetStateCommitment extends EthereumCommitment {
       /// keep in sync with `digest2` definition
       return keccak256(
         solidityPack(
-          ["bytes1", "bytes32", "uint256", "uint256", "bytes32", "bytes1"],
```

Comment:
@IIIIllllIIIIllllIIIIllllIIIIllllIIIIll Please review

Reply:
lgtm

Reply:
extracting this to https://github.com/counterfactual/monorepo/pull/621 since I forsee myself editing this file